morphers/nn.py

```diff
@@ -11,6 +11,23 @@ def __init__(self, dim):
     def forward(self, x):
         return x.unsqueeze(self.dim)
 
+# Time2Vec Module
+class Time2Vec(nn.Module):
+    def __init__(self, size: int):
+        super().__init__()
+
+        self.size = size
+
+        self.weight = nn.Parameter(torch.zeros([1, self.size]))
+        nn.init.kaiming_uniform_(self.weight, nonlinearity="relu")
+        self.bias = nn.Parameter(torch.zeros([1, self.size]))
```

Comment:
It probably makes sense to rename these to something like `frequency` and `phase`, just so it's clearer what's going on.